MonoGame.Framework/Input/GamePad.UWP.cs

```diff
@@ -11,6 +11,11 @@ static partial class GamePad
     {
         internal static bool Back;
 
+        private static int PlatformGetMaxIndex()
+        {
+            return 16;
+        }
+
         private static GamePadCapabilities PlatformGetCapabilities(int index)
         {
             if (index >= WGI.Gamepad.Gamepads.Count)
```

Comment:
Bad idead, it would cause code like:
```c#
GamePad.GetState(PlayerIndex.Two)
```
to cause a crash when only 1 GamePad is connected.